### ISSUE TITLE
Official helm chart 0.6.0 refers to old/missing docker images

### ISSUE THREAD

**Issue by JulianBlin** [NordicWayInterchange/interchange#370]
TLDR; the `example_values.yml` needs an update and napcore-server docker image cannot be found.

---
The official helm chart (located [here](https://console.cloud.google.com/artifacts/docker/nw-shared-w3ml/europe-west4/nordic-way-interchange/helm%2Finterchange)) for 0.6.0 (from 2023-11-28) seems a bit **outdated** for several reasons : 

1. pods listed in `example_values.yml` are using `074e6bd` tag for all the docker images. However, this `074e6bd` commit is from 2019
![image](https://github.com/NordicWayInterchange/interchange/assets/10685391/86cdf58c-a56c-465f-83bd-72d031d6cb4a)

 whereas the Readme of this project says : 
> Federation docker images: All the docker images specified in the github https://github.com/NordicWayInterchange/interchange/ is automatically built by GitHub and **published to the container registry europe-west4-docker.pkg.dev/nw-shared-w3ml/nordic-way-interchange/ on each commit**. Each component in the system has its own registry.
All the images are tagged with git commit hash, and branch name. **Branch "federation-master" is considered to be the stable branch.**

2. there is a napcore_server pod referring to a napcore-server docker image :

```yaml
napcore_server:
  replicas: 1
  image:
    repository: eu.gcr.io/nordic-way-aad182cc/napcore-server
    tag: "074e6bd"
  logging:
  - logger_name: .root
    level: info
secret_dev_no:
  key_store: "test"
```

However, the [official repository](eu.gcr.io/nordic-way-aad182cc/napcore-server) seems to be empty : 
![image](https://github.com/NordicWayInterchange/interchange/assets/10685391/1c4b3d9b-2973-43f4-aa99-933d4e5e8ad5)

Is there any reason for this ?
Should we be able to find this `napcore-server`  image somewhere else ?

3. There seems to be two different docker registries living at the same time : 
- https://console.cloud.google.com/gcr/images/nordic-way-aad182cc
- https://console.cloud.google.com/artifacts/docker/nw-shared-w3ml/europe-west4/nordic-way-interchange

I guess the second one is the right one to use ? If yes, the `example_values.yml` should be changed to use the "new" docker registry and most probably refer to a tag like `federation-master` like explained in the README ? Currently they only have the commit hash, for some of them.

**Comment by kyrrek** [comment]
Hello, and sorry for the late reply.
We have updated the example_values.yml file in the https://github.com/NordicWayInterchange/interchange/tree/example-values branch, and hope to get this updated in the upcoming couple of weeks. Please be adviced that the version you are using is quite old, and quite a lot of changes have been done since then, so upgrading a running cluster might require some work.
